front/src/modules/ui/menu-item/internals/components/StyledMenuItemBase.tsx

```diff
@@ -66,8 +66,12 @@ export const StyledMenuItemBase = styled.li<MenuItemBaseProps>`
 export const StyledMenuItemLabel = styled.div<{ hasLeftIcon: boolean }>`
   font-size: ${({ theme }) => theme.font.size.sm};
   font-weight: ${({ theme }) => theme.font.weight.regular};
+  overflow: hidden;
   padding-left: ${({ theme, hasLeftIcon }) =>
     hasLeftIcon ? '' : theme.spacing(1)};
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 100%;
```

Comment:
Could you also remove this line as I mentioned in my previous comment? The `width: 100%` is making the "No owner" avatar shrink to 14x16 instead of 16x16.

<img width="95" alt="image" src="https://github.com/twentyhq/twenty/assets/3098428/91390284-f235-4f0e-919c-d23061e212da">

Reply:
Done

Reply:
If you look at the line I commented, I meant the `width: 100%` from `StyledMenuItemLabel`, not `StyledMenuItemLeftContent`. Removing it from `StyledMenuItemLeftContent` still makes the checkmark and "No owner" avatar shrink.

I added a commit to fix this. Thank you!